### ISSUE TITLE
MQTT as a source / sink

### ISSUE THREAD

**Issue by mdlmarkham** [mage-ai/mage-ai#3876]
I'm working with IOT applications and integration to enterprise / cloud IT systems - MAGE is fantastic, but it would be even better if it could interact with MQTT data.

**Comment by wangxiaoyou1993** [comment]
Thank you for the feature request @mdlmarkham ! When do you need this feature by?

**Comment by mdlmarkham** [comment]
I'm testing solutions now - don't need anything in production in the short term, but it would be great to know if this is something that you'd consider and if so approximately where it would fit into your roadmap.

**Comment by wangxiaoyou1993** [comment]
@mdlmarkham yes, it definitely fits into our roadmap. What solution are you testing now?

**Comment by mdlmarkham** [comment]
Looking at [HiveMQ](https://www.hivemq.com/), [EMQx](https://www.emqx.io/) and [AWS IoT Core](https://aws.amazon.com/iot-core/) - All three use MQTT V5

**Comment by thaddeuscleo** [comment]
Hello, I'm curios will this feature be develop?

**Comment by jules-ch** [comment]
I'm working on the streaming source. It's working, submitting a PR soon.
